charts/stable/nextcloud/templates/_configmap.tpl

```diff
@@ -27,7 +27,6 @@
 {{/* Define the configmap */}}
 {{- define "nextcloud.configmaps" -}}
 {{- $fullname := (include "tc.v1.common.lib.chart.names.fullname" $) -}}
-{{- $fqdn := (include "tc.v1.common.lib.chart.names.fqdn" $) -}}
```

Comment:
I've no idea why this has to be removed and the PR description is vague at best

Reply:
Because it is not working here.
My application name is `nextcloud2` not `nextcloud`. 

But this function uses the chart name, not the application name. And the resolved FQDN is `nextcloud.ix-nextcloud2` which is wrong for `NX_ONLYOFFICE_NEXTCLOUD_INTERNAL_URL `.

Reply:
So the solution should be to fix fqdn reference

Reply:
https://github.com/truecharts/library-charts/pull/693